corehq/apps/domain/views/internal.py

```diff
@@ -260,6 +261,31 @@ def page_context(self):
         }
 
 
+class ProjectLimitsView(BaseAdminProjectSettingsView):
+    urlname = 'internal_project_limits_summary'
+    page_title = ugettext_lazy("Project Limits")
+    template_name = 'domain/admin/project_limits.html'
+
+    @method_decorator(require_superuser)
```

Comment:
nit: rather than overriding a method just to place the decorator you can put it at the class level:
```python
@method_decorator(require_superuser, name='dispatch')
class ProjectLimitsView(BaseAdminProjectSettingsView):
```